src/assets/i18n/en.json5

```diff
@@ -582,6 +582,23 @@
 
   "bitstream.edit.notifications.error.format.title": "An error occurred saving the bitstream's format",
 
+  "bitstream.edit.form.iiifLabel.label": "IIIF Label",
+
+  "bitstream.edit.form.iiifLabel.hint": "Canvas label for this image",
```

Comment:
The hints never are displayed in the form.

Reply:
It appears that hints weren't working because the component was using the `DynamicInputModel` constructor and not `DsDynamicInputModel` which extends it. Hints now display for iiif fields and for the file name and description fields.

The spacing issue fixed by adding a top-margin to the "label" in `ds-dynamic-form-control-container.component.scss`. This is is going to affect any component that relies on the container. I tried to find another usage but could not. Still, adding a bit of padding the the label seems like something that's always helpful...

label {
  margin-top: calc(0.7 * var(--ds-content-spacing));
}

Canvas width is now updating.